opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -121,8 +160,8 @@ message LabelValue {
   bool has_value = 2;
 }
 
-// A timestamped measurement.
-message Point {
+// Int64Value is a timestamped measurement of int64 value.
+message Int64Value {
   // start_time_unixnano is the time when the cumulative value was reset to zero.
   // This is used for Counter type only. For Gauge the value is not specified and
```

Comment:
will it help to have different data types for gauge and counter then? May be another place to save on some perf and memory

Reply:
On the wire there is no memory saved because a default fixed64 is not sent (0 value). Only internal memory may decrease, I think it is a small RAM saving.

Reply:
Yeah, I had that version initially (with separate messages for guage and counter), we discussed with with @bogdandrutu and decided that small memory saving is not worth the added number of messages that the user need to choose correctly from. IMO it is the right tradeoff, the simplicity of the proto and small number of messages is important.